app/controllers/simplified-account/settings/team-members/team-members.controller.test.js

```diff
@@ -1,104 +1,88 @@
-const proxyquire = require('proxyquire')
 const sinon = require('sinon')
 const { expect } = require('chai')
 const User = require('@models/User.class')
 const userFixtures = require('@test/fixtures/user.fixtures')
+const ControllerTestBuilder = require('@test/test-helpers/simplified-account/controllers/ControllerTestBuilder.class')
 
 const ACCOUNT_TYPE = 'test'
 const SERVICE_ID = 'service-id-123abc'
 
-let req, res, responseStub, getServiceUsersStub, getInvitedUsersStub, teamMembersController
-
-const getController = (stubs = {}) => {
-  return proxyquire('./team-members.controller', {
-    '@utils/response': { response: stubs.response },
-    '@services/user.service':
-      { getServiceUsers: stubs.getServiceUsers, getInvitedUsers: stubs.getInvitedUsers }
-  })
-}
-
-const setupTest = (method, additionalReqProps = {}, additionalStubs = {}) => {
-  responseStub = sinon.spy()
-  const adminUser = new User(userFixtures.validUserResponse({
-    external_id: 'user-id-for-admin-user',
+const adminUser = new User(userFixtures.validUserResponse({
```

Comment:
Why are is the User being initialised with `userFixtures.validUserResponse` rather than the [userData](https://github.com/alphagov/pay-selfservice/blob/master/app/models/User.class.js#L38)?

Reply:
The existing user fixture provides some default values so it seemed sensible to reuse that rather than create a new user from scratch.

Reply:
ah - but I see what you mean, there are some default values in userData too. I did try using this but hit a problem where it didn't recognise the service roles. I can go back and work through it if you think it's better using userData.

Reply:
I think is okay for now, as it's used in loads of places. No need to change.